app/views/search/results.html.erb

```diff
@@ -64,7 +64,8 @@
     <% if @filters.present? %>
       <aside class="col1q filter-container">
         <div id="filters">
-          <h2>Available filters</h2>
+          <h2 class="hd-3">Filter your results</h2>
+          <h3 class="hd-4"><em><%= @pagination[:hits] %> items</em></h3>
```

Comment:
The hits value is a bit confusing for larger sets as displayed here.

Search for "organic chemistry" and it displays `10000 items`. It should probably display something something more like `10,000+ items`. I think we should expect TIMDEX API/OpenSearch to report "more than 10000 items" as "10000" and adjust the UI to anticipate that.

If you want to handle that in a separate ticket that is fine with me.